client/state/themes/selectors/is-requesting-theme.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Internal dependencies
+ */
+import 'state/themes/init';
+
+/**
+ * Returns true if a request is in progress for the specified site theme, or
+ * false otherwise.
+ *
+ * @param  {object}  state  Global state tree
+ * @param  {number}  siteId Site ID
+ * @param  {number}  themeId Theme ID
+ * @returns {boolean}        Whether request is in progress
+ */
+export function isRequestingTheme( state, siteId, themeId ) {
```

Comment:
Probably 🙂 It's not entirely equivalent, because the original method could return `null` or `undefined` if that was the value in `themeId`, but let's assume the result should be a boolean 😄